syntax = "proto3";
package permission;





/*
DeletePermission请求
*/
message DeletePermissionRequest {
    /*
    需要鉴权的动作, 全局唯一
    */
    string action = 1 ;
}

/*
DeletePermission返回
*/
message DeletePermissionResponse {
message Data {
    /*
    删除权限数量
    */
    int32 count = 1 ;
}
        
    
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    结果信息
    */
    string msg = 2 ;
    /*
    数据
    */
    Data data = 3 ;
}

/*
DeletePermissionApi返回
*/
message DeletePermissionResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    permission.DeletePermissionResponse data = 4 ;
}

